### DISCUSSION TITLE
New EverForest Ant Dark Theme

### DISCUSSION THREAD

**Discussion by Lolabird** [TriliumNext/Trilium#4869] [Show and tell]
Hi everyone! I made a new theme for Trilium called [EverForest Ant Dark ](https://github.com/Lolabird/everforest-ant-dark-trilium-theme) to match [Everforest](https://github.com/Fausto-Korpsvart/Everforest-GTK-Theme) and [Ant Dark](https://github.com/EliverLara/Ant/tree/master/kde/Dark) desktop themes. It features:
* Scrollable tables with sticky headers on both axes
* Compatibility with [Zen Mode](https://github.com/Nriver/awesome-trilium/issues/44) (with a few style changes from the original for better accessibility) and [Syntax Highlight](https://github.com/antoniotejada/Trilium-SyntaxHighlightWidget) widgets
* Vertical floating buttons that don't overlap with note content as much

![image](https://github.com/user-attachments/assets/5c823ccc-2eb8-4c92-a6de-5d012f3d5ef4)

For anyone using [Stellar Dark](https://github.com/Lolabird/stellar-dark-theme-trilium), I've also introduced the above features into that theme as well, plus some other much needed cosmetic changes. I hope you all enjoy them!

![image](https://github.com/user-attachments/assets/e5d25905-8727-4419-b69f-12485761cb3f)

**Comment by Nriver** [comment]
Cool theme! Thanks for sharing. Added to https://github.com/Nriver/awesome-trilium

**Reply by Lolabird** [reply]
Thanks @Nriver!
